test/e2e_node/gcloud/gcloud.go

```diff
@@ -136,6 +136,16 @@ func (gc *gCloudClientImpl) Run(
 	go func() {
 		// Create the tmp directory
 		out, err := gc.Command("mkdir", "-p", tDir)
+
+		// Work around for gcloud flakiness - TODO: debug why gcloud sometimes cannot find credentials for some hosts
```

Comment:
Yeah, maybe we will need to resort to ssh tokens, though the k8s e2e tests use gcloud and do not seem to encounter this.  I am hoping we may be able to debug the issue and continue to use gcoud.